tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
   "compilerOptions": {
     "target": "es2015",
-    "module": "commonjs",
+    "module": "nodenext",
+    "moduleResolution": "nodenext",
+    "esModuleInterop": true,
```

Comment:
`esModuleInterop=true` is now the default: https://www.typescriptlang.org/tsconfig/#esModuleInterop

We still set it in our config file so that other tools (eslint, ts-jest) are aligned with it.